### PR TITLE
Add deletehash to database

### PR DESCRIPTION
**Pull request by sandlerben**

### CODE REVIEW

**Comment by sandlerben** [comment]
Once edit-reports incorporates the latest from master, this PR should be more more readable

**Comment by sandlerben** [comment]
This code isn't yet ready for a review. I need to test it first.

File: requirements/common.txt [hunk 1]
```diff
@@ -26,7 +26,7 @@ itsdangerous==0.24
 webassets==0.10.1
 wsgiref==0.1.2
 requests==2.8.1
-WTForms-Components==0.9.9
-pytimeparse==1.1.5
 redis==2.10.5
 rq==0.5.6
+WTForms-Components==0.9.9
```
Comment (rileynwong) [inline requirements/common.txt:32]: nit: if you're feelin nitpicky i might checkout this file since this change is unnecessary

Comment (sandlerben) [inline requirements/common.txt:32]: I just moved it because there was a merge conflict

**Comment by sandlerben** [comment]
@nancywong does it look good now?

**Comment by rileynwong** [comment]
Is something weird going to happen since you merged in master even though this branch was checked out from `edit-reports`? Or will the changes consolidate when you merge this and then merge `edit-reports` back into master?

**Comment by sandlerben** [comment]
> Or will the changes consolidate when you merge this and then merge edit-reports back into master?

Yes. Or I can merge in this code directly.

**Pull request closed without merging**